### PR TITLE
fix: missing closing tags for div and ul in index.html

### PR DESCRIPTION
**Pull request by CalCorbin**
**Description:**
This PR captures a couple very minor bugs in the `index.html` file. There is also one more bug that is similar, but I'll let you find it! :D

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,8 +8,9 @@ <h1 id="title">Jerry Lawson</h1>
   <body id="body">
     <main id="main">
       <div id="img-div">
-      <h6>"The Black video game engineer who pioneered the way we play."</h6>
-      <img id="image" src="https://thecurrent-online.com/wp-content/uploads/2022/12/Jerry_lawson_ca_1980.png" alt="Jerry Lawson pic">
+        <h6>"The Black video game engineer who pioneered the way we play."</h6>
+        <img id="image" src="https://thecurrent-online.com/wp-content/uploads/2022/12/Jerry_lawson_ca_1980.png" alt="Jerry Lawson pic">
+      </div>
```
Comment (CalCorbin) [inline index.html:13]: I added a closing `</div>` tag here to make sure we close the `<div>` on line 10.

File: index.html [hunk 2]
```diff
@@ -20,7 +21,8 @@ <h2> A Timeline of Jerry Lawson's life </h2>
           <li> In the 1970s, his interest in computing encouraged him into Silicon Valley's Homebrew Computer Club, where he was the only black member at the time and eventually crossed paths with Steve Jobs and Steve Wozniak. He referred to Jobs as a business-minded "sparkplug" and was unimpressed when he interviewed Wozniak for a job.
           <li> In the mid-1970s, Lawson helped create the Fairchild Channel F, which was produced in 1976 by Fairchild Semiconductor, where he worked as director of engineering and marketing.
           <li> His work allowed people to play a variety of games in their homes and paved the way for systems, such as the Atatri 2600, Nintendo, Xbox and Playstation.
-          <li> On April 9, 2011, Jerry Lawson died in Mountain View, California due to complications relating to his diabetes. He was survived by his wife, Catherine, and his two children. 
+          <li> On April 9, 2011, Jerry Lawson died in Mountain View, California due to complications relating to his diabetes. He was survived by his wife, Catherine, and his two children.
+        </ul>
```
Comment (CalCorbin) [inline index.html:25]: I added a closing`</ul>` tag here to make sure we close the `<ul>` tag on line 17.

Comment (Npcodes1) [reply index.html:25]: Can't believe I miss those! Thanks!

Comment (Npcodes1) [reply index.html:25]: I forgot closing </li> tags!

**Review by Npcodes1 — APPROVED** [review]

**Pull request merged**